app/championship/battle_results.py

```diff
@@ -0,0 +1,2 @@
+def results(members: dict) -> dict:
```

Comment:
You could place this function and the one from the `Battle` class in a separate file and name it e.g. `battle_functions.py`.